webapp/src/Service/ImportExportService.php

```diff
@@ -542,58 +538,56 @@ public function getResultsData(int $sortOrder, bool $full = false): array
                     }
                     $rank = $ranks[$numPoints];
                 }
-                $awardString = 'Ranked';
+                if ($numPoints === $lowestMedalPoints) {
+                    $awardString = 'Highest Honors';
+                } elseif ($numPoints === $lowestMedalPoints - 1) {
+                    $awardString = 'High Honors';
+                } else {
+                    $awardString = 'Honors';
+                }
             } else {
                 $awardString = 'Honorable';
-                $rank        = '';
+                $rank        = null;
             }
 
-            $groupWinner = "";
+            $groupWinner = null;
             $categoryId  = $teamScore->team->getCategory()->getCategoryid();
             if (!isset($groupWinners[$categoryId])) {
                 $groupWinners[$categoryId] = true;
                 $groupWinner               = $teamScore->team->getCategory()->getName();
             }
```

Comment:
```suggestion
            if (isset($groupWinners[$categoryId])) {
                // Winner was encountered in earlier iteration.
                $groupWinner = null;
            } else {
                $groupWinners[$categoryId] = true;
                $groupWinner               = $teamScore->team->getCategory()->getName();
            }
```

Reply:
It had too many negations for me, this adds 1 extra line and is easier for me to read.